acme/resource_acme_certificate_test.go

```diff
@@ -594,6 +673,45 @@ func TestAccACMECertificate_noDomains(t *testing.T) {
 	})
 }
 
+type testAccCheckACMECertificateStandardOpts struct {
```

Comment:
Note: I'll eventually clean up the rest of the tests using this standard framework. But for now I just want to merge this and move on to the next release :wink: